src/clevertap.js

```diff
@@ -41,6 +48,7 @@ export default class CleverTap {
   #isSpa
   #previousUrl
   #boundCheckPageChanged = this.#checkPageChanged.bind(this)
+  #isDisableWebPopUpSpamControl
```

Comment:
optional: can `isDisableWebPopUpSpamControl` be better named as `isWebPopUpSpamControlDisabled` ?

Reply:
updated to isWebPopUpSpamControlDisabled